app/jobs/arcgis_token_job.rb

```diff
@@ -0,0 +1,35 @@
+class ArcgisTokenJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    analytics.idv_arcgis_token_job_started
+    token_keeper.retrieve_token
+    return true
+  rescue StandardError => e
```

Comment:
Are there specific exceptions we are trying to avoid? We have alerts for when jobs fail, and rescuing in this manner means we won't get the alert.

Reply:
@mitchellhenke that's what i was thinking, not sure how good job deal with it, will remove it then.

Reply:
Removed.